README.md

```diff
@@ -1 +1,144 @@
-# ruby-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [🎯 Catalog of my things](#-catalog-of-my-things)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Test](#test)
+  - [👥 Authors ](#-authors-)
+    - [Author](#author)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [👋 Show your support ](#-show-your-support-)
+  - [🔭Acknowledgments ](#acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 🎯 Catalog of my things<a name="about-project"></a>
+
+> This is a console application that is built with ruby. It is used to manage items of an individual in a catalog. It has functionalities that include adding a book, music and game, with different properties attached to these items.
+>
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby</summary>
+  
+</details>
+
+<!-- prsentation vidéo -->
+### Video presentation 
+[Video](https://drive.google.com/file/d/1w_kzPfXvjtkez7Z3yPGWOS_uLTW-9eGy/view?usp=sharing)
+
+<!-- Features -->
```

Comment:
- [ ] Please make sure the max length of the presentation video is 5 minutes as requested in the project requirements. Currently, your presentation video is 6+ minutes long. Please fix this issue.

![image](https://github.com/rachaouldbabaali/ruby-capstone/assets/83803414/749e680c-6871-4f65-9b35-4586b0e88544)